db/re/item_reform.yml

```diff
@@ -3564,301 +3564,301 @@ Body:
          - Material: Soft_Silk_Cloth
            Amount: 100
         ResultItem: Baksojin_Hat_LT
-        ChangeRefine: -5
```

Comment:
![sus](https://user-images.githubusercontent.com/3517879/204310730-a48b880d-9fed-4353-84aa-0635e1e36425.jpg)
Below here

Reply:
Checked, this is just a problem with github way of displaying the change